vendor/engines/saml_authentication/lib/saml_authentication/engine.rb

```diff
@@ -14,6 +14,8 @@ class Engine < Rails::Engine
       ViewHook.add_hook "devise.sessions.new",
                         "before_login_form",
                         "saml_authentication/sessions/new"
+
+      OneLogin::RubySaml::Logging.logger.level = Logger::DEBUG
```

Comment:
Are you planning to actually merge this and then take it out once the epic is complete, or...?

Reply:
I'm not sure yet. I'm thinking I might leave it in at least through testing on stage. Or we could do something like `OneLogin::RubySaml::Logging.logger.level = Rails.env.production? Logger::INFO : Logger::DEBUG`.

Reply:
Yeah I'm pretty okay with this ternary as a concept, that seems reasonably practical.